src/components/mobile-menu-levels/types.ts

```diff
@@ -0,0 +1,6 @@
+import { ReactNode } from 'react'
+
+export interface MobileMenuLevelData {
+	levelButtonText: string
+	content: ReactNode
+}
```

Comment:
❓ Just a general question: do you still like defining types like this in separate files? I know its been the convention, but curious to hear your opinion.

Reply:
Great question! Yeah this was more habit than anything else 👍 I'd definitely be on board with being less prescriptive about a separate `types.ts` for each and every component. Sometimes it feels useful... but at the same time, I think when types for the props API of a component get complex enough that they feel useful to put in a separate file, that's probably a sign we should break down the component a bit or something 😅